requirements.txt

```diff
@@ -6,7 +6,7 @@ halo
 tqdm
 deepdiff
 
-fhir.resources==7.1.0  # FHIR Model
+fhir.resources==8.0.0b4  # FHIR Model Pre-release
```

Comment:
Why do we need to update fhir.resources? If we don't need to do so, I think we should separate it out from this ticket, as it could have unknown uninteded consequences for data submissions, hence why #110 is its own ticket imo

Reply:
Yeah that can get reverted